apps/charge-accounts-service/src/operators/operators.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Body, Controller, Get, Post, UseGuards } from '@nestjs/common'
+import { User } from '@app/accounts-service/users/user.decorator'
+import { UsersService } from '@app/accounts-service/users/users.service'
+import { JwtAuthGuard } from '@app/accounts-service/auth/guards/jwt-auth.guard'
+import { ProjectsService } from '@app/accounts-service/projects/projects.service'
+import { CreateOperatorDto } from '@app/accounts-service/operators/dto/create-operator.dto'
+
+@Controller({ path: 'operators', version: '1' })
+export class OperatorsController {
+  constructor (
+    private readonly usersService: UsersService,
+    private readonly projectsService: ProjectsService
+  ) {}
+
+  /**
+   * Get current operator
+   * @param authOperatorDto
+   * @returns the user and project with public key
+   */
+  @UseGuards(JwtAuthGuard)
+  @Get('/me')
```

Comment:
Please delete '/me' from this request it be much cleaner to

Reply:
@NikolayRn GET /operators means fetching all operators, but here we only need to fetch one operator, which also needs to be the requester and authorized.

Discord also uses the /me endpoint for the same purpose:
https://discord.com/developers/docs/resources/user#get-current-user

Reply:
@MusabShakeel576 Youre right about all operators, however "/me"  in the context of retrieving a specific operator's data, using /operators/{id} would be more explicit and align with RESTful API conventions.

Reply:
@NikolayRn renamed /me to /operators/{id}